src/clr-angular/utils/focus/key-focus/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Type } from '@angular/core';
+
+import { ClrKeyFocus } from './key-focus';
+import { ClrKeyFocusItem } from './key-focus-item';
+
+export * from './key-focus';
+
+export const KEY_FOCUS_DIRECTIVES: Type<any>[] = [ClrKeyFocus, ClrKeyFocusItem];
```

Comment:
Define these in the module file and only re-export in the index file. We never want to import a defined item from an index file.